app.rb

```diff
@@ -4,26 +4,34 @@
 require_relative "list"
 
 def debug_params
-  puts "PARAMS: #{params}"
+  puts "reload page: #{params}"
 end
 
 get "/" do
   # HINT: you can use instance variables in the view directly without passing to locals
   # such as this @title instance variable
-  @title = "Your App Name"
-  list = List.new("0")
-  list.load_from_file
-  erb :"index.html", locals: {list: list}, layout: :"layout.html"
+  @title = "Sunny TODO"
+  #list = List.new("0")
+  #list.load_from_file
+  lists = List.load_all_lists
```

Comment:
Nice job refactoring to `List.load_all_lists`! Please delete all the other comments in this action because you don't need them any more